AppCenter/AppCenterTests/MSChannelGroupDefaultTests.m

```diff
@@ -29,14 +30,15 @@ - (void)setUp {
                                                               flushInterval:flushInterval
                                                              batchSizeLimit:batchSizeLimit
                                                         pendingBatchesLimit:pendingBatchesLimit];
+  [MSAuthTokenContext sharedInstance].authToken = nil;
 }
 
 #pragma mark - Tests
 
 - (void)testNewInstanceWasInitialisedCorrectly {
 
   // If
-  id ingestionMock = OCMProtocolMock(@protocol(MSIngestionProtocol));
+  MSAppCenterIngestion *ingestionMock = OCMPartialMock([MSAppCenterIngestion new]);
```

Comment:
Any reason to change to a partial mock?

Reply:
Agreed, why?

Reply:
Yes there is a reason - technically, we were passing `id<MSIngestionProtocol>` to a method that expects `MSAppCenterIngestion *`, and this was not noticed until now because we are now actually using a method in the latter that doesn't exist in the former.